code/__HELPERS/icons.dm

```diff
@@ -1328,3 +1328,54 @@ GLOBAL_LIST_EMPTY(transformation_animation_objects)
 	final_color_code[2] = FLOOR(final_color_code[2]/total, 1)
 	final_color_code[3] = FLOOR(final_color_code[3]/total, 1)
 	return "#[num2hex(final_color_code[1], 2)][num2hex(final_color_code[2], 2)][num2hex(final_color_code[3], 2)]"
+
+/*
+ * Make a copy of a human and copy their appearance without copying any special overlays. Notice that this currently doesn't include held items!
+ */
+/mob/proc/CopyHumanAppearance(mob/target)
```

Comment:
Why is this a proc on the mob level? Is there ANY situation where ghosts/undead mobs should be using this
If not, i'd recommend putting it on the abnormality zone

Reply:
Probably not, but there isn't really a good reason to put it at the abnormality level either. There's always adminbus, or if someone wants to make a distortion that is some sort of doppelganger. I'm more concerned about how it currently  doesn't grab held items, but for the purpose of this PR specifically, that doesn't matter.

Reply:
Eh, if someone wants to make an abno that uses this they can always bump it down a level, imo this is unnecessary